cmd/podman/generate/systemd.go

```diff
@@ -85,6 +86,10 @@ func systemd(cmd *cobra.Command, args []string) error {
 		logrus.Warnln("The generated units should be placed on your remote system")
 	}
 
+	if !systemdOptions.New && systemdOptions.TemplateUnitFile {
+		return errors.New("--template requires --new")
```

Comment:
I don't like this requirement.  Podman should be smart enough to automatically set `--new=false` unless the user explicitly changed it `podman generate systemd --template --new=false`.

Reply:
Note: once changed, please update the man page.